VaticanBAV/et/BAVet115.xml

```diff
@@ -797,49 +790,49 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                           <title type="complete" ref="LIT1931Mashaf#SecondBlessing">Second blessing of ʾAbbā Sāmuʾel</title>
                            <incipit xml:lang="gez">
                               ካልዕ፡ ቡራኬ፡ ላዕለ፡ ማኅበረ፡ ቅዱሳን፡ ወጽውዓን፡ ወለክሙኒ፡ አበውየ፡ ወአኃውየ፡ ያክህልክሙ፡ እግዚአብሔር፡ ለገቢረ፡ ፈቃዱ፡ ወሥምረቱ፡ 
                               ወይኅድር፡ መንፈስ፡ ቅዱስ፡ ላዕሌክሙ፡ ወይኅፅርክሙ፡ በሐፁረ፡ መስቀሉ፡
                            </incipit>
                         </msItem>
                         <msItem xml:id="ms_i1.42.3">
                            <locus from="133v" to="134r"/>
-                           <title type="complete" ref="LIT1931Mashaf#ThirdBlessing">Third blessing of ʾabbā Sāmuʾel</title>
+                           <title type="complete" ref="LIT1931Mashaf#ThirdBlessing">Third blessing of ʾAbbā Sāmuʾel</title>
                            <incipit xml:lang="gez">
                               ሣልስ፡ ቡራኬ፡ ኦደቂቅየ፡ ጽውዓን፡ ዘጸውዓክሙ፡ እግዚአብሔር፡ ለገቢረ፡ ሥም<supplied reason="omitted">ረ</supplied>ቱ፡ ይኅድር፡ 
                               ላዕሌክሙ፡ መንፈስ፡ <supplied reason="omitted">ቅዱስ፡</supplied> ወይቀድስክሙ፡
                            </incipit>
                         </msItem>
                         <msItem xml:id="ms_i1.42.4">
                            <locus from="134r" to="134v"/>
-                           <title type="complete" ref="LIT1931Mashaf#FourthBlessing">Fourth blessing of ʾabbā Sāmuʾel</title>
+                           <title type="complete" ref="LIT1931Mashaf#FourthBlessing">Fourth blessing of ʾAbbā Sāmuʾel</title>
                            <incipit xml:lang="gez">
                               ራብዕ፡ ቡራኬ፡ ተልዕሎ፡ ያልዕልክሙ፡ እግዚአብሔር፡ በመንግሥተ፡ ሰማያት፡ በከመ፡ ቃሉ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ 
                               <del rend="expunctuated">በ</del>ዘኢይኄሱ፡ በወንጌል፡ ቅዱስ፡ ንዑ፡ ኀቤየ፡ ቡሩካኑ፡ ለአቡየ፡ ትረሱ፡ መንግሥተ፡ ሰማያት፡
                            </incipit>
                         </msItem>
                         <msItem xml:id="ms_i1.42.5">
                            <locus target="#134v"/>
-                           <title type="complete" ref="LIT1931Mashaf#FifthBlessing">Fifth blessing of ʾabbā Sāmuʾel</title>
+                           <title type="complete" ref="LIT1931Mashaf#FifthBlessing">Fifth blessing of ʾAbbā Sāmuʾel</title>
                            <incipit xml:lang="gez">
                               ኃምስ፡ ቡራኬ፡ ለዛቲ፡ ቤተ፡ ክርስቲያን፡ ያጽንዓ፡ ወይ<supplied reason="omitted">ዕ</supplied>ቀባ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እስከ፡ ዳግም፡ 
                               ምጽአቱ፡ እንዘ፡ ይትገበር፡ ውስቴታ፡ ሕግ፡ ወሥርዓት፡
                            </incipit>
                         </msItem>
                         <msItem xml:id="ms_i1.42.6">
                            <locus from="134v" to="135r"/>
-                           <title type="complete" ref="LIT1931Mashaf#SixthBlessing">Sixth blessing of ʾabbā Sāmuʾel</title>
+                           <title type="complete" ref="LIT1931Mashaf#SixthBlessing">Sixth blessing of ʾAbbā Sāmuʾel</title>
```

Comment:
Massimo, numbers of blessings, are they stable elsewhere and belong to the work? if they are, than i agree with your way of rendering them. If, however, numbers serve rather as navigating elements (like first miracle, second etc), than I think you need not to correct the numeration, but just make a note, stating that number 5 is repeated.